call-a-function-api-oci-clouddev.md

```diff
@@ -1,5 +1,4 @@
 ---
-layout: collection
```

Comment:
You're still letting this one happen. `layout: collection` will 100% break any post you put it on. Leave `layout` off completely, or at worst, set it to `layout: single`.